tests/Common/TriggersTest.php

```diff
@@ -492,31 +493,51 @@ public function testDeleteTriggerWithMasterToken()
             ],
         ]);
 
-        $loadedTrigger = $this->_client->getTrigger((int) $trigger['id']);
-        $this->assertEquals($trigger['id'], $loadedTrigger['id']);
+        $trigger2 = $this->_client->createTrigger([
+            'component' => 'orchestrator',
+            'configurationId' => 123,
+            'coolDownPeriodMinutes' => 10,
+            'runWithTokenId' => $newToken['id'],
+            'tableIds' => [
+                $table,
+            ],
+        ]);
 
-        $newNonAdminToken = $this->tokens->createToken((new TokenCreateOptions())->setCanManageBuckets(true));
-        $clientWithoutAdminToken = $this->getClient(['url' => STORAGE_API_URL, 'token' => $newNonAdminToken['token']]);
+        $newNonAdminTokenWithoutPermission = $this->tokens->createToken((new TokenCreateOptions()));
+        $clientWithoutAdminTokenWithoutPermission = $this->getClient(['url' => STORAGE_API_URL, 'token' => $newNonAdminTokenWithoutPermission['token']]);
```

Comment:
Tohle je dost divný pattern v kombinaci s tím data providerem - vlastně to otestuješ 4x pro každý z těch dataproviderů :/ 

Ale už to nechme. 

Ideálně by to bylo:

`testTriggerCreatedByMasterTokenCanBeDeletedByToken` (tady by byly ty 4 tokeny co to s nima jde)
`testTriggerCreatedByMasterTokenCannotBeDeletedByToken` (tady by byl ten jeden co to s ním nejde)